Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -143,18 +145,28 @@ template <typename sp_range_t, typename out_range_t>
 void SeedFinder<external_spacepoint_t, platform_t>::getCompatibleDoublets(
     const Acts::SeedFinderOptions& options, sp_range_t& otherSPs,
     const InternalSpacePoint<external_spacepoint_t>& mediumSP,
-    out_range_t& outVec, const float& deltaRMinSP, const float& deltaRMaxSP,
-    bool isBottom) const {
+    out_range_t& outVec, std::vector<LinCircle>& linCircleVec,
+    const float& deltaRMinSP, const float& deltaRMaxSP, bool isBottom) const {
   const int sign = isBottom ? -1 : 1;
 
   outVec.clear();
+  linCircleVec.clear();
+
+  // reserve enough capacity
+  size_t dubletCapacity =
+      isBottom ? m_config.bottomDubletCapacity : m_config.topDubletCapacity;
+  outVec.reserve(dubletCapacity);
+  linCircleVec.reserve(dubletCapacity);
```

Comment:
I would not go for this approach here... This is highly arbitrary since it depends on what kind of event you have in your hands. 

Also, we already know that the maximum capacity of these vectors is the size of `otherSPs`, right?